src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -190,7 +193,7 @@ def __init__(self, path, security_scope=None, secure=True):
 
 
 def Endpoint(method=None, path=None, path_params=None, query_params=None,  # noqa: N802
-             json_response=True, proxy=False, xml=False):
+             json_response=True, proxy=False, xml=False, version=None, exclude_ver=False):
```

Comment:
Is it really necessary to add a new input arg? Did my suggestion not work?:
- `version undefined` =>  `version = DEFAULT_VERSION`
- `version = None` => version is disabled

I think `None` allows for handling the no-version use case.

Reply:
Yes, will work once the version parameter is set to DEFAULT_VERSION. I did this to make it work with the current default value we pass in paramter.

Reply:
So, are you keeping the exclude_ver here or getting rid of it?

Reply:
Will get rid of it!